frontend/www/js/omegaup/components/arena/ContestListv2.vue

```diff
@@ -203,9 +203,7 @@
                   <font-awesome-icon icon="calendar-alt" />
                   <a :href="getTimeLink(contestItem.finish_time)">
                     {{
-                      ui.formatString(T.contestEndTime, {
-                        endDate: finishContestDate(contestItem),
-                      })
+                     finishContestDate(contestItem)
```

Comment:
As mentioned in the previous comment, it would be worth keeping these text strings localized so that they respect the preferred language of the logged-in user:
```suggestion
                      ui.formatString(T.contestEndTime, {
                        endDate: finishContestDate(contestItem),
                      })
```